src/App.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import logo from './logo.svg'
+import './App.css'
+import LoginOrSignup from './Pages/Portal/LoginOrSignup'
```

Comment:
Try using modules. Create a `index.ts` in `Pages` and exprot stuff from there so you can write `import { LoginOrSignup | from './Pages`